conda/testing/helpers.py

```diff
@@ -20,18 +20,18 @@
 from time import time
```

Comment:
Those moves make me happy!

Reply:
We need to clean some unused symbols but I'll do that on a coming PR!